src/include/trusted_ca_keys.h

```diff
@@ -0,0 +1,45 @@
+// Copyright 2023 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef TRUSTED_CA_KEYS_H_
+#define TRUSTED_CA_KEYS_H_
+
+#include <vector>
+#include <fstream>
+#include <string>
+
+using std::string;
+using std::vector;
+using std::ofstream;
+
+// A default API endpoint
+const char *mds_url_def = "http://169.254.169.254/computeMetadata/v1/oslogin/certificates";
+// A default file with CA keys
+const char *trusted_keys_file_def = "/etc/ssh/trustedca.pub";
+// A suffix for the temporary file
+const char *tmp_suff = ".temp";
```

Comment:
What do you suggest? To use `#define` or hardcode?
I don't think both are suitable here.
I used a const because it is much easier to maintain such values in the future and I concatenate physical entities of clear types.